configs/fashion/2d_kpt_sview_rgb_img/topdown_heatmap/deepfashion2/resnet_deepfashion2.md

```diff
@@ -0,0 +1,67 @@
+<!-- [ALGORITHM] -->
+
+<details>
+<summary align="right"><a href="http://openaccess.thecvf.com/content_ECCV_2018/html/Bin_Xiao_Simple_Baselines_for_ECCV_2018_paper.html">SimpleBaseline2D (ECCV'2018)</a></summary>
+
+```bibtex
+@inproceedings{xiao2018simple,
+  title={Simple baselines for human pose estimation and tracking},
+  author={Xiao, Bin and Wu, Haiping and Wei, Yichen},
+  booktitle={Proceedings of the European conference on computer vision (ECCV)},
+  pages={466--481},
+  year={2018}
+}
+```
+
+</details>
+
+<!-- [BACKBONE] -->
+
+<details>
+<summary align="right"><a href="http://openaccess.thecvf.com/content_cvpr_2016/html/He_Deep_Residual_Learning_CVPR_2016_paper.html">ResNet (CVPR'2016)</a></summary>
+
+```bibtex
+@inproceedings{he2016deep,
+  title={Deep residual learning for image recognition},
+  author={He, Kaiming and Zhang, Xiangyu and Ren, Shaoqing and Sun, Jian},
+  booktitle={Proceedings of the IEEE conference on computer vision and pattern recognition},
+  pages={770--778},
+  year={2016}
+}
+```
+
+</details>
+
+<!-- [DATASET] -->
+
+<details>
+<summary align="right"><a href="https://arxiv.org/pdf/1901.07973.pdf">DeepFashion2 (CVPR'2019)</a></summary>
+
+```bibtex
+@article{DeepFashion2,
+  author = {Yuying Ge and Ruimao Zhang and Lingyun Wu and Xiaogang Wang and Xiaoou Tang and Ping Luo},
+  title={A Versatile Benchmark for Detection, Pose Estimation, Segmentation and Re-Identification of Clothing Images},
+  journal={CVPR},
+  year={2019}
+}
+```
+
+</details>
+
+Results on DeepFashion2 val set
+
+| Set                   |                        Arch                         | Input Size | PCK@0.2 |  AUC  | EPE  |                        ckpt                         |                         log                         |
+| :-------------------- | :-------------------------------------------------: | :--------: | :-----: | :---: | :--: | :-------------------------------------------------: | :-------------------------------------------------: |
+| short_sleeved_shirt   | [pose_resnet_50](/configs/fashion/2d_kpt_sview_rgb_img/topdown_heatmap/deepfashion2/res50_deepfashion2_short_sleeved_shirt_256x192.py) |  256x256   |  0.988  | 0.703 | 10.2 | [ckpt](https://drive.google.com/file/d/1waYaWYPXum9Lncv4qmd0H7lt8Abgb0Ry/view?usp=share_link) | [log](https://drive.google.com/file/d/1RCsyvHDkAiA_SMwLoQ9YUvgq8jrCcmUm/view?usp=share_link) |
```

Comment:
Some links of the log seems to be incorrect. For example, the log link for _short_sleeved_shirt_ is the same as that for _long_sleeved_shirt_ .

Copy-pasting the link is error-prone. How about organizing these files into a directory, and simply share the link of the directory to us?

Reply:
Ok, I will do that

Reply:
I put all log in and models in this [link](https://drive.google.com/drive/folders/1AfYT_NOpBXHYmdDKjKjXcIJj8rYz_Ylb), it can also be found in the commit.

Reply:
One more thing left, we will use this [publish_model tool](https://github.com/open-mmlab/mmpose/blob/master/tools/misc/publish_model.py) to rename the model ckpts before release. 
Could you help run this tool and follow our naming rules to rename the model ckpts and logs?

For example:
ckpt name:  res50_deepfashion_upper_256x192-41794f03_20210124.pth
log name: res50_deepfashion_upper_256x192_20210124.log.json

`res50_deepfashion_upper_256x192` is the same as the config .py
`41794f03` is sha256sum automatically generated by the publish_model tool.
`20210124` is the date automatically generated by the publish_model tool.

reference: https://raw.githubusercontent.com/open-mmlab/mmpose/master/configs/fashion/2d_kpt_sview_rgb_img/topdown_heatmap/deepfashion/resnet_deepfashion.md

Reply:
> One more thing left, we will use this [publish_model tool](https://github.com/open-mmlab/mmpose/blob/master/tools/misc/publish_model.py) to rename the model ckpts before release. Could you help run this tool and follow our naming rules to rename the model ckpts and logs?
> 
> For example: ckpt name: res50_deepfashion_upper_256x192-41794f03_20210124.pth log name: res50_deepfashion_upper_256x192_20210124.log.json
> 
> `res50_deepfashion_upper_256x192` is the same as the config .py `41794f03` is sha256sum automatically generated by the publish_model tool. `20210124` is the date automatically generated by the publish_model tool.
> 
> reference: https://raw.githubusercontent.com/open-mmlab/mmpose/master/configs/fashion/2d_kpt_sview_rgb_img/topdown_heatmap/deepfashion/resnet_deepfashion.md

I will try

Reply:
usage:

`python tools/misc/publish_model.py work_dirs/res50_deepfashion2_long_sleeved_dress_256x192/best_PCK_epoch_190.pth res50_deepfashion2_long_sleeved_dress_256x192`

Reply:
I have uploaded renamed models to google drive. All links has been changed to the directory link for convenience.

Reply:
@WangXin93  Thanks a lot! The PR looks good to me. We will merge it soon.

@ly015 Could you help make a backup copy of these model ckpts and logs in our openmmlab's cloud storage space?